test/dotnet-publish.Tests/PublishTests.cs

```diff
@@ -294,5 +294,15 @@ public void PublishAppWithOutputAssemblyName()
             var command = new TestCommand(Path.Combine(publishedDir.FullName, outputExe));
             command.Execute("").Should().ExitWith(0);
         }
+
+        [Fact]
+        public void PublishFailsWhenProjectRootIsEmpty()
```

Comment:
Since you've added two failure cases (when a full path is passed, and when a directory is passed) do you think it'd make sense to have tests for both?